rpcserver/rpcserver.cc

```diff
@@ -131,6 +132,20 @@ void RPCServer::handle_rpc(tcp::socket sock) {
     }
 }
 
+/**
+ * Register all the rpc functions handlers (currently just client to namenode) 
+ */
+void RPCServer::register_handlers() {
+	using namespace client_namenode_translator;
+
+	register_handler("getFileInfo",	 		ClientNamenodeTranslator::getFileInfo);
```

Comment:
The server shouldn't know what handlers are going to be registered. We want to move this over to the rice-namenode main.cc